code/modules/VR/vr_goggles.dm

```diff
@@ -0,0 +1,44 @@
+//Glorified teleporter that puts you in a new human body.
+// it's """VR"""
+
+/obj/item/clothing/glasses/vr_goggles
+	desc = "A long time ago, people used these glasses to makes images from screens threedimensional."
+	name = "3D glasses"
+	icon_state = "3d"
+	item_state = "3d"
+	prescription_upgradable = 0
+	species_fit = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/species/vox/eyes.dmi'
+		)
+
+	var/you_die_in_the_game_you_die_for_real = FALSE
+	var/mob/living/carbon/human/virtual_reality/vr_human
+
+
+/obj/item/clothing/glasses/vr_goggles/Destroy()
+	..()
+	vr_human.revert_to_reality(1)
+	cleanup_vr_avatar()
+
+/obj/item/clothing/glasses/vr_goggles/emag_act(mob/user)
+	//placeholder
```

Comment:
That's not code, that's a comment - as in, not even a functional statement when uncommented in any sense of the term